utilities/ideascale-importer/ideascale_importer/snapshot_importer/importer.py

```diff
@@ -233,11 +266,27 @@ async def _write_db_data(self):
 
             await asyncio.sleep(0)
 
+        is_snapshot_final = False
+        should_update_final = False
+
+        if self.lastest_block_time is None or self.snapshot_start_time is None:
+            logger.warning("Could not determine if snapshot is final, final flag will not be updated")
+        elif self.lastest_block_time is not None and \
+                self.snapshot_start_time is not None:
+            is_snapshot_final = self.lastest_block_time >= self.snapshot_start_time
+            should_update_final = True
+            logger.info(
+                "Setting snapshot final flag",
+                final=is_snapshot_final,
+                snapshot_start=self.snapshot_start_time.isoformat(),
+                latest_block_time=self.lastest_block_time.isoformat(),
+            )
+
         snapshot = models.Snapshot(
             event=self.event_id,
             as_at=datetime.utcnow(),
             last_updated=datetime.utcnow(),
```

Comment:
last_updated needs to be the time read into `lastest_block_time` which tells us what time dbsync was at when we ran the command.  The time on the local machine isn't important due to dbsync and local time not being synchronized.